test/lib/build/ProjectBuilder.js

```diff
@@ -727,7 +727,7 @@ test("_executeCleanupTasks", async (t) => {
 	const executeCleanupTasksStub = sinon.stub(builder._buildContext, "executeCleanupTasks");
 	await builder._executeCleanupTasks();
 	t.is(executeCleanupTasksStub.callCount, 1, "BuildContext#executeCleanupTasks got called once");
-	t.deepEqual(executeCleanupTasksStub.getCall(0).args, [],
+	t.deepEqual(executeCleanupTasksStub.getCall(0).args, [undefined],
```

Comment:
LGTM, but could you please add another test where we call `_executeCleanupTasks` with `force=true` and test that the callback correctly receives the flag?

Reply:
Done